package.json

```diff
@@ -93,11 +93,15 @@
     "@types/react": "^17",
     "@types/react-dom": "^17",
     "@types/react-router-dom": "^5.1.7",
+    "@typescript-eslint/eslint-plugin": "^5.21.0",
+    "@typescript-eslint/parser": "^5.21.0",
     "browserify-fs": "^1.0.0",
     "browserify-zlib": "^0.2.0",
     "crypto-browserify": "^3.12.0",
+    "eslint": "^8.14.0",
```

Comment:
I'm not 100% if we need to install ESLint here because react-scripts is using ESLint8 under the hood.

Reply:
I'm going to test this locally, and see if it can be removed.

Reply:
removed